rustls/Cargo.toml

```diff
@@ -24,7 +24,7 @@ brotli-decompressor = { version = "4.0.1", optional = true } # 4.0.1 required fo
 hashbrown = { version = "0.14", optional = true, default-features = false, features = ["ahash", "inline-more"] }
 log = { version = "0.4.8", optional = true }
 # remove once our MSRV is >= 1.70
-once_cell = { version = "1.16", default-features = false, features = ["alloc", "race"] }
+once_cell = { version = "1.16", default-features = false, features = ["alloc", "critical-section"] }
```

Comment:
Fails on `thumbv6m` - https://github.com/rustls/rustls/actions/runs/10661757574/job/29547782037?pr=2088#step:5:71

```
error[E0599]: no method named `compare_exchange` found for struct `portable_atomic::AtomicUsize` in the current scope
  --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/once_cell-1.19.0/src/race.rs:60:24
   |
60 |             self.inner.compare_exchange(0, value.get(), Ordering::AcqRel, Ordering::Acquire);
   |                        ^^^^^^^^^^^^^^^^ method not found in `AtomicUsize`
```

Fails in `race.rs` even though `race` features is not requested or needed from `once_cell`.

Reply:
Could you try 

```
once_cell = { version = "1.16", default-features = false, features = ["critical-section"] }
```

That is, without alloc? It seems that the only thing enabled by `alloc` is `race`, so, if you need `race`, you should be able to remove `alloc`?

Reply:
Solved it - thanks & apologies!